returnn/tf/layers/basic.py

```diff
@@ -6158,6 +6159,9 @@ def __init__(
         :param dict[str,str]|None filter_perm: transposes the filter (input filter as layer)
         :param LayerBase|None bias: if given, will not create an own parameter, but use this as the bias
         :param bool use_time_mask:
+        :param Optional[float] pad_seq_len_to_power: pad seq len to power of given number to reduce number of different
+            seq lens. See https://github.com/rwth-i6/returnn/issues/1450 and
+            https://github.com/tensorflow/tensorflow/issues/62441.
```

Comment:
"seq lens" should not be together in the same line with the next sentence, if the next sentence is partially split over again more lines.
```suggestion
        :param Optional[float] pad_seq_len_to_power: pad seq len to power of given number
            to reduce number of different seq lens.
            See https://github.com/rwth-i6/returnn/issues/1450 and
            https://github.com/tensorflow/tensorflow/issues/62441.
```